app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1185,6 +1187,49 @@ class PatientRepositoryAndroidTest {
     }
   }
 
+  @Test
+  fun verify_deleted_Communications_are_not_included_when_fetching_recent_patients() {
```

Comment:
Rename method to use all small cases.
`verify_deleted_communications_are_not_included_when_fetching_recent_patients()`

Reply:
done